src/system/system.md

```diff
@@ -6,7 +6,7 @@ sections:
    url: /system/data-transfer.html
```

Comment:
the tabs seem to have different text formats.  some start with Learn how to....others start with the task (Set Up Admin rights).

Reply:
I think that is kind of a good thing that they are varied a little bit that way... although I would say I don't really like the "learn how to" phrasing for things unless it is a tutorial.